calibration/simulator.py

```diff
@@ -0,0 +1,314 @@
+import numpy as np
```

Comment:
Same thing here, there's almost no change from `aspics/simulator.py`. Let's make this work using the original copy.